streams-contrib/streams-provider-datasift/pom.xml

```diff
@@ -29,6 +29,11 @@
 
     <description>Datasift Provider</description>
 
+    <properties>
+        <skipITs>true</skipITs>
+        <testDataBaseURl>http://streams.peoplepattern.com.s3.amazonaws.com/test-data/</testDataBaseURl>
```

Comment:
Yes - these reference test files can reside in this commerically-hosted repository with public permissions until/unless there is a complaint.  More important than where these specific test files are hosted today, integration tests can now run against real-world data hosted by anyone at any accessible URL.